static/css/devhub/new-landing/common.less

```diff
@@ -88,9 +89,10 @@
 
 .DevHub-callout-box {
   background: #fff;
-  margin: 0 20px 0;
+  margin: 0 0 0;
```

Comment:
Could this be `margin: 0;`

Reply:
https://github.com/mozilla/addons-server/pull/12623/commits/550b10cd3a66c75bba75139e5baea6ea6431a24c